xml_loader/shift_generation.py

```diff
@@ -315,15 +315,16 @@ def already_daily_off_shift(root, employee_offset, employee_rest, day):
 
 def get_shift_lists(root):
 
-    desired_dur = const.DESIRED_SHIFT_DURATION
+    min_shift_dur = min(const.ALLOWED_SHIFT_DURATION)
```

Comment:
Her bør du heller bruke `const.ALLOWED_SHIFT_DURATION[0]`. Tilsvarende gjelder for max.

Reply:
Kan støttes.